doc/classes/RigidBody2D.xml

```diff
@@ -127,7 +127,7 @@
 			<return type="Vector2">
 			</return>
 			<description>
-				Return the body linear velocity. This changes by physics granularity. See [method set_linear_velocity].
+				Return the body's linear velocity. This changes by physics granularity. See [method set_linear_velocity].
```

Comment:
Return -> Returns

Changes by physics granularity how?  What is "physics granularity"?  If you're going to use terms like these in the description, you have to explain them. Remember, this is going to be read by beginners, too.